src/tests/test-lbl_unused_lbls.do

```diff
@@ -36,13 +38,35 @@ label define v4 1 "evet" 2 "hayır", modify
 * attach one value label, but not the others
 label values v1 v1
 
+label list
+
+tempfile lbl_data
+save    `lbl_data', orphans
```

Comment:
I actually did not know this myself until today, but yes, that is that case. While this makes the command perhaps a bit less required, I do not think there is any harm in keeping it. I think it can still be useful to test that one has not accidently made a value label an orphan.

Reply:
Good point.